sdk/go/pulumi-language-go/testdata/projects/l2-resource-simple/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi-simple/sdk/v2/go/simple"
```

Comment:
```suggestion
	"github.com/pulumi/pulumi-simple/sdk/go/v2/simple"
```
Because of changing the go.mod location this should be changed as well. I did wonder if we clean up the nested "simple" folder as well but that makes for horrible import names (you'd have to do `simple "github.com/pulumi/pulumi-simple/sdk/v2/go"` so it doesn't really save you the typing)

Reply:
Done ✅